library/kotlin/src/io/envoyproxy/envoymobile/RetryPolicy.kt

```diff
@@ -0,0 +1,26 @@
+package io.envoyproxy.envoymobile
+
+/**
+ * Specifies how a request may be retried, containing one or more rules.
+ *
+ * @param maxRetryCount Maximum number of retries that a request may be performed.
+ * @param retryOn Whitelist of rules used for retrying.
+ * @param perRetryTimeoutMs Timeout (in milliseconds) to apply to each retry.
+ */
+data class RetryPolicy(
+    val maxRetryCount: Int,
+    val retryOn: List<RetryRule>,
```

Comment:
This should be optional

Reply:
Why not just keep it an empty list? If there's no retry rules, shouldn't it be the same as if it were to be unset?

Reply:
Hmm, that's true. Ok yea - I'll update Swift

Reply:
🤦‍♂ already non-optional in Swift, ignore haha